src/date-range-picker/__tests__/date-range-picker-absolute.test.tsx

```diff
@@ -241,6 +241,34 @@ describe('Date range picker', () => {
       });
     });
 
+    describe('time formats', () => {
+      test('correctly handles a time format of hh', () => {
```

Comment:
How about adding a test for `hh:mm` too? To demonstrate consistency between modes

Reply:
Added for all time formats 👍🏻